website/docs/d/google_billing_account.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "google"
+page_title: "Google: google_billing_account"
+sidebar_current: "docs-google-datasource-billing-account"
+description: |-
+  Get information about a Google Billing Account.
+---
+
+# google\_billing\_account
+
+Use this data source to get information about a Google Billing Account.
+
+```hcl
+data "google_billing_account" "acct" {
+  display_name = "My Billing Account"
+}
+
+resource "google_project" "my_project" {
+  name       = "My Project"
+  project_id = "your-project-id"
+  org_id     = "1234567"
+
+  billing_account = "${data.google_billing_account.acct.id}"
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available billing accounts.
+The given filters must match exactly one billing account whose data will be exported as attributes.
+The following arguments are supported:
+
+* `name` (Optional) - The resource name of the billing account in the form `billingAccounts/{billing_account_id}`.
```

Comment:
Consider doing like we did for folders where you can specify either "folders/{folder_id}" or "{folder_id}".

Since we would support only "{billing_account_id}" for this field, what do you think or renaming this field `account_id` or `billing_account_id`?

https://github.com/terraform-providers/terraform-provider-google/blob/master/google/iam_folder.go#L81

Reply:
How about keeping `name` as a _Computed_-only attribute and having the argument named `billing_account` specified as either `{billing_account_id}` or `billingAccounts/{billing_account_id}`?
GCP seems exact in its distinction between resource names and resource IDs and this argument can be either.

Reply:
Agree. I like your suggestion better.

Reply:
Done.